mmv1/products/apigee/terraform.yaml

```diff
@@ -359,6 +359,50 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       test_env_vars:
         org_id: :ORG_ID
         billing_account: :BILLING_ACCT
+  EnvKeystore: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{env_id}}/keystores/{{name}}", "{{env_id}}/{{name}}"]
+    delete_url: '{{env_id}}/keystores/{{name}}'
+    skip_sweeper: true
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_environment_keystore_test"
+      primary_resource_id: "apigee_environment_keystore"
+      test_env_vars:
+        org_id: :ORG_ID
+        billing_account: :BILLING_ACCT
+      skip_docs: true
+      # Resource creation race
+      skip_vcr: true
+    timeouts: !ruby/object:Api::Timeouts
```

Comment:
Do we need such large timeouts for these resources? These are needed for resources that take a long time to set up

Reply:
No, resources are created instantly and doesn't have major dependency resources to put major timeouts, I ll go with 1 minute